api/pkg/apps/webapp/templates/form_element.gohtml

```diff
@@ -6,14 +6,16 @@
                     class="form-select"
                     aria-label="Default select example"
                     {{if .Attributes.Multiple}}multiple{{end}}
-                    {{if .Validation.Required}}required{{end}}>
+                    {{if .Validation.Required}}required{{end}}
+                    {{if .Attributes.Value}}disabled{{end}}
```

Comment:
Value might not be there if required=false, so disabled should be true when case is not a new case